Dockerfile.ppc64le

```diff
@@ -95,25 +94,15 @@ RUN set -x \
 
 
 # Install Go
-# ppc64le doesn't have official go binaries, so use the version of go installed from the image
-# to build go from source.
-# NOTE: ppc64le has compatibility issues with older versions of go, so make sure the version >= 1.6
+# NOTE: official ppc64le go binaries were added with go 1.7.4, so versions of
+# go < 1.7.4 will have to build go from src using an older version as bootstrap.
+# Also due to compatability issues, ppc64le requires a go version >= 1.6.0,
```

Comment:
Fair point. I don't think it hurts, and I really don't want anyone to downgrade the version to say 1.7.3 and be confused as to why it doesn't work (this has happened a couple of times in the past)